### PR TITLE
add Vancouver

### PR DESCRIPTION
**Pull request by CoderLogy**

### CODE REVIEW

File: .replit
Comment (samdev-7) [inline .replit:1]: Please don't commit this file! Could you revert this change?

File: README.md
Comment (samdev-7) [inline README.md:1]: Please do not modify files that are not your city!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by CoderLogy** [comment]
done @samdev-7 fixed the problems

**Comment by CoderLogy** [comment]
Please let me know

**Review by samdev-7 — APPROVED** [review]

**Comment by CoderLogy** [comment]
Why it got canceled @samdev-7

**Comment by CoderLogy** [comment]
Please approve it

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/vancouver**](https://counterspell-ja08s9y70.hackclub.dev/vancouver) |
|:-:|
|<a href="https://counterspell-ja08s9y70.hackclub.dev/vancouver">Screenshot of /vancouver</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/hzY9oNb3RgSRQa5YFcmWxE5KloDBJPiezcQ9iUapqCyp0C7J0F5z_b3UsB0B2CQDZkQnsLl-vfiCLpPJLS2xfw.png">(view full size)</a> |







Commit 1a054a73d0da1d94d071d2f663d7771487916b49 (https://counterspell-ja08s9y70.hackclub.dev).

**Comment by CoderLogy** [comment]
But it still shows this 
image

**Comment by CoderLogy** [comment]
please merge the repo @samdev-7 @lachlanjc

**Pull request merged**